packages/taiko-client/integration_test/hive_tests.go

```diff
@@ -0,0 +1,36 @@
+package integration_test
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/ethereum/hive/hivesim"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestHiveHandler(t *testing.T) {
+	baseDir := os.Getenv("HIVE_BASE_DIR")
+	if baseDir == "" {
+		t.SkipNow()
+	}
+	handler, err := hivesim.NewHiveFramework(&hivesim.HiveConfig{
+		BaseDir:        baseDir,
+		SimPattern:     "taiko2",
```

Comment:
Let's use `taiko` instead

Reply:
OK, will be fixed in the hive repo and then change here.